config/sync/core.entity_view_display.node.machine.teaser.yml

```diff
@@ -25,14 +25,22 @@ dependencies:
     - field.field.node.machine.field_image_more_materials
     - field.field.node.machine.field_ip
     - field.field.node.machine.field_languages
+    - field.field.node.machine.field_location
     - field.field.node.machine.field_login_allow_manual
     - field.field.node.machine.field_login_allow_scan
+    - field.field.node.machine.field_logstash_host
```

Comment:
Any reason for adding all these other fields to the teaser, besides field_screen_off/on?

Reply:
I have not added them... just an config export form prod database.

Reply:
ok